### ISSUE TITLE
`expiring-todo-comments` Broken in ESlint `9.15.0`

### ISSUE THREAD

**Issue by Lalem001** [sindresorhus/eslint-plugin-unicorn#2496]
`unicorn/expiring-todo-comments` Broken in ESlint `9.15.0`. Works fine in `9.14.0`.

> TypeError: Error while loading rule 'unicorn/expiring-todo-comments': Cannot read properties of undefined (reading 'decoration')

The above error occurs when reading a file that has the following comment:

```js
// Note: If none of the slotted options are selected, select all
```

Relevant portion of ESLint config is:

```js
{
  files: ['**/*'],
  extends: [eslint.configs.recommended, unicorn.configs['flat/recommended']],
  linterOptions: {
    reportUnusedDisableDirectives: true,
  },
},
```

**Comment by Lalem001** [comment]
Relevant dependencies in package.json:

```json
devDependencies: {
  "@eslint/js": "^9.15.0",
  "eslint": "^9.15.0",
  "eslint-plugin-unicorn": "^56.0.0",
}
```

**Comment by marcalexiei** [comment]
Might be related to https://github.com/eslint/eslint/issues/19134

**Comment by Pnlvfx** [comment]
I can confirm, same error after updating to eslint 9.15

**Comment by Pnlvfx** [comment]
typescript-eslint was affected and does fix it, now we have to wait for this

**Comment by ipanasenko** [comment]
They've fixed it in [8.15.0](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v8.15.0)

**Comment by Lalem001** [comment]
> Might be related to [eslint/eslint#19134](https://github.com/eslint/eslint/issues/19134)

ESLint team chalked this one up to an issue with `typescript-eslint` which just pushed an update to fix it on their end.

> They've fixed it in [8.15.0](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v8.15.0)

I updated all my deps and still have the same issue. Disabling unicorn avoids the error. So the issue persists in unicorn.
